openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -151,11 +151,12 @@
 - control_key: SC-4 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: partial
   narrative:
     - text: |
-        This control is met when SELinux is enabled and enforced and through
-        the use of SELinux MCS Levels.
+        The customer is responsible for preventing the unauthorized transfer of
+        shared resources that may be beyond the controls of OpenShift.  Storage
+        choices and procedures related to persistent volumes must be addressed.'
```

Comment:
`SC-4 (2)` is a MLS control. 

```The information system prevents unauthorized information transfer via shared resources in accordance with [Assignment: organization-defined procedures] when system processing explicitly switches between different information classification levels or security categories.```

How is this partial and the customer's responsibility when switching in between SELinux MLS levels? @kevin-anderson-redhat can  you provide some commentary on this? Thanks!